environments/mcdTransformers.toml

```diff
@@ -1,1667 +1,1667 @@
-        "flop_v1_0_9",
-        "jug",
-        "median_bat",
-        "median_comp",
-        "median_eth",
-        "median_knc",
-        "median_link",
-        "median_lrc",
-        "median_mana",
-        "median_usdt",
-        "median_wbtc",
-        "median_zrx",
-        "pot",
-        "spot",
-        "vat",
-        "vow",
+  home = "github.com/makerdao/vulcanizedb"
+  name = "transformerExporter"
+  save = false
+  schema = "maker"
+  transformerNames = ["cat_v1_0_0",
+    "cat_v1_1_0",
+    "cdp_manager",
+    "flap_v1_0_0",
+    "flap_v1_0_9",
+    "flip_bat_a_v1_0_0",
+    "flip_bat_a_v1_0_9",
+    "flip_bat_a_v1_1_0",
+    "flip_comp_a_v1_1_2",
+    "flip_eth_a_v1_0_0",
+    "flip_eth_a_v1_0_9",
+    "flip_eth_a_v1_1_0",
+    "flip_knc_a_v1_0_8",
+    "flip_knc_a_v1_0_9",
+    "flip_knc_a_v1_1_0",
+    "flip_link_a_v1_1_2",
+    "flip_lrc_a_v1_1_2",
+    "flip_mana_a_v1_0_9",
+    "flip_mana_a_v1_1_0",
+    "flip_paxusd_a_v1_1_1",
+    "flip_sai_v1_0_0",
+    "flip_tusd_a_v1_0_7",
+    "flip_tusd_a_v1_0_9",
+    "flip_tusd_a_v1_1_0",
+    "flip_usdc_a_v1_0_4",
+    "flip_usdc_a_v1_0_9",
+    "flip_usdc_a_v1_1_0",
+    "flip_usdc_b_v1_0_7",
+    "flip_usdc_b_v1_0_9",
+    "flip_usdc_b_v1_1_0",
+    "flip_usdt_a_v1_1_1",
+    "flip_wbtc_a_v1_0_6",
+    "flip_wbtc_a_v1_0_9",
+    "flip_wbtc_a_v1_1_0",
+    "flip_zrx_a_v1_0_8",
+    "flip_zrx_a_v1_0_9",
+    "flip_zrx_a_v1_1_0",
+    "flop_v1_0_1",
+    "flop_v1_0_9",
+    "jug",
+    "median_bat",
+    "median_comp",
+    "median_eth",
+    "median_knc",
+    "median_link",
+    "median_lrc",
+    "median_mana",
+    "median_usdt",
+    "median_wbtc",
+    "median_zrx",
+    "pot",
+    "spot",
+    "vat",
+    "vow",
+    "auction_file",
+    "bite",
+    "cat_claw",
+    "cat_file_box",
+    "cat_file_chop_lump_dunk",
+    "cat_file_flip",
+    "cat_file_vow",
+    "deal",
+    "dent",
+    "deny",
+    "flap_kick",
+    "flip_file_cat",
+    "flip_kick",
+    "flop_kick",
+    "jug_drip",
+    "jug_file_base",
+    "jug_file_ilk",
+    "jug_file_vow",
+    "jug_init",
+    "log_median_price",
+    "log_value",
+    "median_diss_batch",
+    "median_diss_single",
+    "median_drop",
+    "median_kiss_batch",
+    "median_kiss_single",
+    "median_lift",
+    "new_cdp",
+    "osm_change",
+    "pot_cage",
+    "pot_drip",
+    "pot_exit",
+    "pot_file_dsr",
+    "pot_file_vow",
+    "pot_join",
+    "rely",
+    "spot_file_mat",
+    "spot_file_par",
+    "spot_file_pip",
+    "spot_poke",
+    "tend",
+    "tick",
+    "vat_deny",
+    "vat_file_debt_ceiling",
+    "vat_file_ilk",
+    "vat_flux",
+    "vat_fold",
+    "vat_fork",
+    "vat_frob",
+    "vat_grab",
+    "vat_heal",
+    "vat_hope",
+    "vat_init",
+    "vat_move",
+    "vat_nope",
+    "vat_rely",
+    "vat_slip",
+    "vat_suck",
+    "vow_fess",
+    "vow_file_auction_attributes",
+    "vow_file_auction_address",
+    "vow_flog",
+    "vow_heal",
+    "yank",
+    "flip_eth_b_v1_1_3"
```

Comment:
ndb but maybe it'd be cool to put this one close to the other flip storage transformers? Looks like we might have to accept some disorder if we're adding these things programmatically, but wondering if it'd be beneficial to generally clean up after ourselves for ease of manual parsing